app/controllers/api/v1/additional_document_validation_requests_controller.rb

```diff
@@ -3,13 +3,33 @@
 module Api
   module V1
     class AdditionalDocumentValidationRequestsController < Api::V1::ValidationRequestsController
-      skip_before_action :verify_authenticity_token, only: :update
+      skip_before_action :verify_authenticity_token
 
-      before_action :check_token_and_set_application,
-                    :check_file_params_are_present,
+      before_action :check_token_and_set_application
+      before_action :check_file_params_are_present,
                     :check_file_size,
                     :check_file_type, only: :update
 
+      def index
+        respond_to do |format|
+          format.json do
```

Comment:
should we default the API routes to `json` format? https://guides.rubyonrails.org/routing.html#defining-defaults

Reply:
ooh nice, haven't seen this before

Reply:
I'll definitely add this but do it in the next PR. Need to add a test to api_index_spec too (but planning to do something clever with Melissa on this).